_includes/search.html

```diff
@@ -0,0 +1,4 @@
+<form class="search__container">
+  <input type="text" name="query" placeholder="Search the docs" class="search__input">
```

Comment:
Any input should have a label, even if not visible to our eyes. The `for` attribute of a label ties it to a specific input.